modules/utilityFunctions.js

```diff
@@ -0,0 +1,36 @@
+import books from './Books.js';
+import * as Elements from './constElements.js';
+
+const uploadContent = () => {
+  Elements.booksList.innerHTML = '';
+  books.BooksObject.forEach((obj) => {
+    Elements.booksList.innerHTML += `<div class="book-container">
+                      <div class="book">
+                        <h4 class="text-1">"${obj.title}"</h4>
+                        <h3 class="text-1">by ${obj.author}</h3>
+                      </div>
+                      <button type="button" class="btn deleteBtn" data-id="${obj.id}">Remove</button>
+                  </div>`;
+  });
+};
+
+function checkLocalStorage() {
+  if (JSON.parse(localStorage.getItem('BOOKS_LIST')) != null) {
+    books.BooksObject = JSON.parse(localStorage.getItem('BOOKS_LIST'));
+    uploadContent();
+  }
+}
```

Comment:
- [ ] according to the requirements we should use ES6 syntax which includes usage of arrow functions instead of the function keyword, please replace this.